cachito/web/content_manifest.py

```diff
@@ -232,3 +237,246 @@ def generate_icm(self, image_contents=None):
         icm = deepcopy(BASE_ICM)
         icm["image_contents"] = image_contents or []
         return deep_sort_icm(icm)
+
+
+class ContentManifestPackage:
+    """
+    A package within a content manifest.
+
+    It is used primarily to generate a package URL (purl).
+    """
+
+    def __init__(self, name, type, version, dev=False, dependencies=[], path=None, replaces=None):
```

Comment:
Mutable defaults (the list) are usually a [bad idea](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments), typically one would use `None` instead

Reply:
Oh shit, I did not know that :open_mouth: 

That's what you get when you think you're smarter than your IDE :see_no_evil: